tests/queries/0_stateless/02968_mysql_show_warnings.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+# Tag no-fasttest: requires mysql client
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+${MYSQL_CLIENT} --execute "SHOW WARNINGS;"
+${MYSQL_CLIENT} --execute "show warnings;"
+${MYSQL_CLIENT} --execute "SHOW WARNINGS LIMIT 100;"
+${MYSQL_CLIENT} --execute "show warnings limit 100;"
+${MYSQL_CLIENT} --execute "SHOW WARNINGS LIMIT 100 OFFSET 100;"
+${MYSQL_CLIENT} --execute "show warnings limit 100 offset 100;"
+${MYSQL_CLIENT} --execute "SHOW COUNT(*) WARNINGS;"
+${MYSQL_CLIENT} --execute "show count(*) warnings;"
+${MYSQL_CLIENT} --execute "SELECT @@session.warning_count;"
```

Comment:
`@@session.warnings_count` is a [system variable](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_warning_count) in MySQL. My feeling is that re-implementing (even just faking) these in ClickHouse opens a can of worms.

If you did not see `SELECT @@session.warning_count;` in log files, I'd prefer to not provide compatibility with `@@session.warning_count`.

Reply:
It already works even without this PR. I only added it to the test cause it's mentioned in the docs.